tests/testthat/test-06-calibrate.R

```diff
@@ -23,8 +23,8 @@ test_that("calibrate can set language", {
   ## With french translation
   expect_error(run_calibrate(model_output, list(opt = "options"), tempdir(),
                              language = "fr"),
-               paste0("La sortie du modèle n'est pas à jour. Veuillez ",
-                      "relancer le modèle et réessayer."))
+               paste0("La sortie du modèle est obsolète, ",
```

Comment:
This needed updating after https://github.com/mrc-ide/naomi/pull/375/files#diff-58c0c0dd3bace2e632c5d2ccc23e5ac09c2ba5a8051741c720957021108e22feR57 has been merged